code/frameworks/angular/package.json

```diff
@@ -20,18 +20,18 @@
     "url": "https://opencollective.com/storybook"
   },
   "license": "MIT",
-  "main": "dist/types/index.js",
-  "module": "dist/types/index.js",
-  "types": "dist/types/index.d.ts",
+  "main": "dist/index.js",
```

Comment:
No module?

Reply:
Yes, we have no ESM build for angular (yet). So I guess better not to pretend :D